pkg/anchor/writer/writer.go

```diff
@@ -350,26 +353,33 @@ func (c *Writer) signCredentialWithLocalWitnessLog(vc *verifiable.Credential) (*
 
 	logger.Debugf("sign credential with local witness: %s", string(vcBytes))
 
+	witnessStartTime := time.Now()
```

Comment:
I have a feeling that Witness will need more granular metrics since it's doing a lot.

Reply:
yes I will add it in separate PR